repo/packages/S/spark/12/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "packagingVersion": "2.0",
```

Comment:
Do you need to update this to 3.0 and add "minDcosReleaseVersion": "1.8"?
Maybe unnecessary for Argus, since we'll just package both Spark/HDFS into a local universe and push that out, but we can't merge into universe master with these changes that won't(?) work in 2.x